### PR TITLE
Travis CI

### PR DESCRIPTION
**Pull request by raphamorim**
Hi @gnarf, @leobalter
I gave the kickoff for integration. I wonder if can enable jquery-color in travis.

Currently the script only runs grunt, but I'll integrate it with browserstack later.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+sudo: false
+language: node_js
+node_js:
+  - "0.10"
```
Comment (leobalter) [inline .travis.yml]: we might use a newer version here.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+before_install:
+  - npm install -g grunt-cli
```
Comment (leobalter) [inline .travis.yml]: might be improved later, but we can use grunt-cli from a local node_module. It's a ++ if we have it on this PR

File: package.json [hunk 3]
```diff
@@ -8,6 +8,9 @@
     "name": "jQuery Foundation and other contributors",
     "url": "https://github.com/jquery/jquery-color/blob/master/AUTHORS.txt"
   },
+  "scripts": {
+    "ci": "grunt"
```
Comment (leobalter) [inline package.json]: grunt default task is defined as:

```js
grunt.registerTask( "default", [ "jshint", "qunit", "build", "compare_size" ] );
```

maybe we could just call `grunt jshint qunit` here.

Comment (gnarf) [inline package.json]: I'd prefer `grunt ci` and registering that task

File: .travis.yml (hunk 1 above)
Comment (raphamorim) [inline .travis.yml]: Update: b562f600a374f6431bc8d7d4fbe0f2851ff0b624

File: package.json (hunk 3 above)
Comment (leobalter) [inline package.json]: yes yes

File: .travis.yml (hunk 1 above)
Comment (leobalter) [inline .travis.yml]: we don't need to have multiple Node versions, we're not testing anything on Node, maybe the only issue we might have is due to dependencies, but we don't use anything else than devDeps so far.

File: package.json (hunk 3 above)
Comment (raphamorim) [inline package.json]: Updated: 44858fc0836512b53bb2a239bb806b3c7e5f71db

File: .travis.yml (hunk 2 above)
Comment (raphamorim) [inline .travis.yml]: Updated: e54b4ab27c1d73e42545997255455d9261404168

File: package.json [hunk 4]
```diff
@@ -25,6 +28,7 @@
   "devDependencies": {
     "grunt": "0.4.5",
     "grunt-bowercopy": "1.0.0",
+    "grunt-cli": "^0.1.13",
```
Comment (leobalter) [inline package.json]: the other packages are version locked. We should lock it too for consistency.

File: .travis.yml (hunk 2 above)
Comment (leobalter) [inline .travis.yml]: :clap: :clap:

File: .travis.yml (hunk 1 above)
Comment (raphamorim) [inline .travis.yml]: Gotcha, so just node `5`?

File: package.json (hunk 4 above)
Comment (raphamorim) [inline package.json]: Updated: 228b1dcfa72d682b93803e042c5501dc97622234

File: .travis.yml (hunk 1 above)
Comment (raphamorim) [inline .travis.yml]: Updated: 228b1dcfa72d682b93803e042c5501dc97622234

**Comment by leobalter** [comment]
LGTM.

**Comment by leobalter** [comment]
@raphamorim would you squash the commits and set a commit message based on the [jQuery Commit guidelines](http://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines)?

You may use _Build_ as the component name.

**Comment by leobalter** [comment]
One last thing: it seems you might need to run `bower install`

**Comment by raphamorim** [comment]
I don't know about it, because jquery-color uses [Bowercopy](https://www.npmjs.com/package/grunt-bowercopy). Then by default, bowercopy runs `bower install`.

**Comment by gnarf** [comment]
the current processes here were copied from a REALLLY old version of the
jquery build chain.

everything about it could use "simplifying" probably (including bowercopy,
we can npm jquery now anyway)

**Comment by raphamorim** [comment]
I see, 

I think it's better merge this PR ( 'cause probably shouldn't change the script) and open an issue to discuss the build improvements.

What do you think @gnarf, @leobalter ?

**Comment by leobalter** [comment]
I'm ok with the "progressive enhancement".

I just want to turn on Travis CI before I merge this.

**Comment by leobalter** [comment]
merged at 44cbc46

:+1:

**Pull request closed without merging**